docs/serving/configuration/deployment.md

```diff
@@ -106,3 +109,27 @@ data:
   # List of repositories for which tag to digest resolving should be skipped
   registries-skipping-tag-resolving: registry.example.com
 ```
+
+## Enable Freezer Service
+
+You can configure queue-proxy to call a endpoint when it's traffic drops to zero or scales up from zero by modifying `concurrency-state-endpoint` ConfigMap setting. With this configuration, you can achieve some features like freezing runing process in user-contianer or charing calculation based on time of handling requests。
```

Comment:
```suggestion
You can configure queue-proxy to call an endpoint when its traffic drops to zero or scales up from zero by modifying the `concurrency-state-endpoint` ConfigMap setting. With this configuration, you can achieve some features like freezing a running process in a user-container or charing calculation based on time of handling requests.
```

Reply:
I'm not sure what is meant by: 
> charing calculation based on time of handling requests.

Could you clarify this please?

Reply:
fixed, I change it to `billing based on the time it takes to process the request`